lib/run-script-pkg.js

```diff
@@ -4,6 +4,10 @@ const packageEnvs = require('./package-envs.js')
 const { isNodeGypPackage, defaultGypInstallScript } = require('@npmcli/node-gyp')
 const signalManager = require('./signal-manager.js')
 const isServerPackage = require('./is-server-package.js')
+const { ShellString, unquoted } = require('puka');
+
+const sh = (...args) =>
+  ShellString.sh(...args).toString(process.env.__FAKE_TESTING_PLATFORM__ || process.platform)
```

Comment:
`puka` comes with a `sh` export (defined as `const sh = (...args) => ShellString.sh(...args).toString()` basically), but we’re making our own so we can test the Windows version more easily.

https://gitlab.com/rhendric/puka#shellstring

---

`process.env.__FAKE_TESTING_PLATFORM__ || process.platform` is copied from is-windows.js – let me know if you’d like to share that, and if so, how.